pkg/minio/minio_test.go

```diff
@@ -39,7 +39,7 @@ import (
 const (
 	objectName string = "test.yaml"
 	objectEtag string = "2020beab5f1711919157756379622d1d"
-	region     string = "us-east-1"
+	region     string = "us-west-2"
```

Comment:
I am confused why I need this but I see the same errors in both local and GH actions like https://github.com/fluxcd/source-controller/runs/6316538345?check_suite_focus=true#step:7:67

Reply:
`play.min.io` seems to have moved to a different region. I had to add the same change on another open PR.

Reply:
That change has now been merged, so this is no longer needed here. :+1: